.github/workflows/agora-docker.yml

```diff
@@ -0,0 +1,51 @@
+name: Build agora
+
+on: [push, pull_request]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        if: github.event_name == 'push'
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          submodules: false
```

Comment:
That looks wrong

Reply:
> That looks wrong

I need to check out the agora repo to execute the `git describe`, otherwise I get 'no .git repository found'. What is your suggestion?

Reply:
I mean, this specific line, have `submodules: false`. Why ?

Reply:
> I mean, this specific line, have `submodules: false`. Why ?

We don't need the submodules to execute git describe. Anyways if I try to put the Version number together from github env variables as you suggested, then I don't need to checkout anything.

Reply:
But you do need  the submodules to build Agora. You will always need to `checkout`, otherwise there's nothing to build.

Reply:
it turned out the 'build docker' github action did checkout all the source, so this step can be removed